### DISCUSSION TITLE
Album item count is different from original on phone

### DISCUSSION THREAD

**Discussion by tam481** [immich-app/immich#14936] [Q&A]
Hello
I uploaded an album with 652 items from my Android phone via the Immich app. The app recognises the number of files as 652 in the album when selecting, however, it says that it has uploaded everything (Remainder is showing 0 assets to upload) and the album in Immich both in the app and in the web view is showing 647 items. There are no "other" files. The gallery app is reporting 570 images and 82 videos.

How can I check what is going on and more importantly, which files have not been uploaded?

**Comment by alextran1502** [comment]
There are probably duplicated files. We don't show them but you can check the duplicated count in App Settings > Advanced > Clear Duplicated
